torch/onnx/symbolic_opset9.py

```diff
@@ -1351,6 +1351,14 @@ def zeros_like(g, input, dtype=None, layout=None, device=None, pin_memory=False,
                 value_t=torch.tensor([0], dtype=sym_help.scalar_type_to_pytorch_type[dtype]))
 
 
+@parse_args('v', 'v', 'i', 'v', 'v', 'v')
```

Comment:
I am wondering what's appropriate annotation for layout, device, and pin_memory.